zipkin-server/src/main/java/zipkin2/server/internal/elasticsearch/ZipkinElasticsearchStorageProperties.java

```diff
@@ -64,8 +64,9 @@
    */
   private int timeout = 10_000;
 
-  public ZipkinElasticsearchStorageProperties(@Value("${zipkin.storage.throttle.enabled:false}") boolean throttleEnabled,
-                                              @Value("${zipkin.storage.throttle.maxConcurrency:200}") int throttleMaxConcurrency) {
+  ZipkinElasticsearchStorageProperties(
+    @Value("${zipkin.storage.throttle.enabled:false}") boolean throttleEnabled,
+    @Value("${zipkin.storage.throttle.maxConcurrency:200}") int throttleMaxConcurrency) {
```

Comment:
Does `maxConcurrency` need updated to `max-concurrency` here?

Reply:
shouldn't be required, but it is nice to do this. good catch

(ps I'm knackered so will deal with the other PR and any comments in the morning)

Reply:
Kk.  Thank you for finishing this up and sorry again for the lag on my end!  Work got in the way :(